app/locales/fr.json

```diff
@@ -1,87 +1,87 @@
 {
 	"commonText": {
 		"appName": "Forest Watcher 2.0",
-		"next": "Next",
-		"back": "Back",
-		"finish": "Finish",
-		"setUp": "Set up",
-    "cancel": "Cancel",
-    "confirm": "Confirm"
+		"next": "Suivant",
+		"back": "Retourner",
+		"finish": "Terminer",
+		"setUp": "Installer",
+    "cancel": "Annuler",
+    "confirm": "Confirmer"
 	},
 	"dashboard": {
-		"title": "Dashboard",
-		"alerts": "See my alerts",
-		"reports": "Make a report",
-		"myReports": "My reports",
-		"uploadData": "Saved and uploaded data",
-		"map": "Map"
+		"title": "Panel",
+		"alerts": "Voir mes alertes",
+		"reports": "Faire un rapport",
+		"myReports": "Mes rapports",
+		"uploadData": "Données sauvegardées et téléchargées",
```

Comment:
Données sauvegardées will do both. French doesn't have a word to say "upload". "Saved" is sufficient I think.